common/src/main/java/tds/common/logging/EventLogger.java

```diff
@@ -19,6 +19,12 @@
  * Centralized event logger.
  */
 public class EventLogger {
+
+    public enum LogEvent {
+        APP_STARTUP,
```

Comment:
I know. The issue here is that there's an EventLogger for legacy apps, and one for all the new microservices. This one here is shared among everyone who uses TDS_Common, so that's good.